Bindings/Packages/com.virtualmaker.bindings/Runtime/VirtualMaker.Bindings.asmdef

```diff
@@ -11,6 +11,27 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": [],
+    "versionDefines": [
+        {
+            "name": "com.unity.ugui",
+            "expression": "",
```

Comment:
I don't think we should leave these blank. set min or range of versions to enable it for.